Assets/VRTK/Source/Scripts/Utilities/SDK/VRTK_SDKManager.cs

```diff
@@ -818,8 +818,10 @@ static VRTK_SDKManager()
             PopulateAvailableScriptingDefineSymbolPredicateInfos();
             PopulateAvailableAndInstalledSDKInfos();
 
-#if UNITY_EDITOR
+#if UNITY_EDITOR && UNITY_2018_1_OR_NEWER
```

Comment:
I feel it could be easier to just nest `#if's here, rather than doing the `#if`-`#elif`.